test/customer_test.rb

```diff
@@ -3,22 +3,26 @@
 require_relative '../lib/customer'
 
 class CustomerTest < MiniTest::Test
+  attr_reader :parent, :customer
+  def setup
+    sales_engine = SalesEngine.new
+    @parent = CustomerRepository.new('test/support/sample_customers.csv', sales_engine)
+    @customer = Customer.new({:id => "3", :first_name => "Mariah", :last_name => "Toy", :created_at => "2012-03-27 14:54:10 UTC", :updated_at => "2012-03-27 14:54:10 UTC"},  parent)
```

Comment:
Line is too long. [181/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.